pkg/authentication/scram/mock_types_test.go

```diff
@@ -1,65 +0,0 @@
-package scram
```

Comment:
Should these tests go into the new `mocks` package or were they obsoleted by this patch?

Reply:
They weren't tests, they were methods for the mock types. Everything in this file was moved to util/mock_types.go